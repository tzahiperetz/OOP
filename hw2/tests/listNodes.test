# listNodes test

# Create a graph
CreateGraph g1
CreateGraph g2

# Add a pair of nodes
CreateNode n1 1
CreateNode n2 2
CreateNode n3 3
CreateNode n4 4
CreateNode n5 5
CreateNode n6 6

AddNode g1 n1
AddNode g1 n2
AddNode g1 n3
AddNode g1 n4
AddNode g1 n5
AddNode g1 n6
AddNode g2 n1
AddNode g2 n2

# Add an edge
AddEdge g1 n1 n2
AddEdge g1 n1 n3
AddEdge g1 n1 n4
AddEdge g1 n1 n5
AddEdge g1 n1 n6
AddEdge g1 n2 n3
AddEdge g1 n2 n4
AddEdge g1 n2 n5
AddEdge g1 n2 n6
AddEdge g1 n4 n3
AddEdge g1 n4 n2
AddEdge g1 n4 n1
AddEdge g1 n1 n2
AddEdge g1 n2 n1

ListNodes g1
ListNodes g2
