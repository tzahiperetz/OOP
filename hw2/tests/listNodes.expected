# listNodes test

# Create a graph
created graph g1
created graph g2

# Add a pair of nodes
created node n1 with cost 1
created node n2 with cost 2
created node n3 with cost 3
created node n4 with cost 4
created node n5 with cost 5
created node n6 with cost 6

added node n1 to g1
added node n2 to g1
added node n3 to g1
added node n4 to g1
added node n5 to g1
added node n6 to g1
added node n1 to g2
added node n2 to g2

# Add an edge
added edge from n1 to n2 in g1
added edge from n1 to n3 in g1
added edge from n1 to n4 in g1
added edge from n1 to n5 in g1
added edge from n1 to n6 in g1
added edge from n2 to n3 in g1
added edge from n2 to n4 in g1
added edge from n2 to n5 in g1
added edge from n2 to n6 in g1
added edge from n4 to n3 in g1
added edge from n4 to n2 in g1
added edge from n4 to n1 in g1
failed to add edge
added edge from n2 to n1 in g1

g1 contains: n1 n2 n3 n4 n5 n6
g2 contains: n1 n2
